arch/xtensa/src/esp32/esp32_psram.c

```diff
@@ -1530,7 +1530,7 @@ psram_enable(int mode, int vaddrmode)   /* psram init */
         PSRAM_IS_64MBIT_TRIAL(s_psram_id)))
     {
 #if defined(CONFIG_ESP32_SPIRAM_2T_MODE)
-#  if CONFIG_ESP32_SPIRAM_BANKSWITCH_ENABLE
+#  if CONFIG_SPIRAM_BANKSWITCH_ENABLE
```

Comment:
CONFIG_SPIRAM_BANKSWITCH_ENABLE or CONFIG_ESP32_SPIRAM_BANKSWITCH_ENABLE?
In arch/xtensa/src/esp32/Kconfig, this configure option changed to ESP32_SPIRAM_BANKSWITCH_ENABLE

Please kindly check.

Thanks